Gulpfile.js

```diff
@@ -30,8 +30,9 @@ var saveLicense = require('uglify-save-license');
 var tag = require('./bin/tag');
 var runSequence = require('run-sequence');
 
-gulp.task('test-server-start', function () {
+gulp.task('test-server-start', function (done) {
     server.run(['test/server.js']);
+    setTimeout(done, 1000);
```

Comment:
Looks like there's a potential race condition here on travis where the server hasn't started before the mocha tests run - this just gives it a little delay.